src/index.html

```diff
@@ -10,8 +10,195 @@
+        <legend>Registration</legend>
+        <label for="email">
+          E-mail:
+        </label>
+        <input
+          class="form-controls"
+          type="email"
+          placeholder="email@example.com"
+          name="email"
+          id="email"
+          required>
+        <br>
+        <label for="password">
+          Password:
+        </label>
+        <input
+          type="password"
+          name="password"
+          minlength="8"
+          id="password"
+          required
+        >
+      </fieldset>
+
+      <fieldset class="field-group">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-controls">
+          <label for="fact">
+            Do you love cats?
+          </label>
+          <input
+            type="radio"
+            name="yes"
+            id="yes"
+            value="yes">
+          <label for="yes">
+            Yes
+          </label>
+          <input
+            type="radio"
+            name="no"
+            id="no"
+            value="no"
+          >
+          <label for="no">
+            No
+          </label>
+        </div>
+
+        <div class="form-controls">
+          <label for="color">
+            What is your favorite color?
+          </label>
+          <input
+            type="color"
+            id="color">
+        </div>
+
+        <div class="form-controls">
+          <label for="time">
+            What time do you go to bed?
+          </label>
+          <input
+            type="time"
+            id="time">
+        </div>
+
+        <div class="form-controls">
+          <label for="cars">
+            What are your favorite brand of cars?
+          </label>
+          <select
+            name="cars"
+            id="cars"
+            multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </div>
+
+
+        <label for="rating">
+          How do you rate our work?
+        </label>
+        <input
+          type="range"
+          name="range"
+          value="0"
+          id="rating"
+        >
+      </div>
+      </fieldset>
+
+      <fieldset class="field-group">
+        <legend>Additional info:</legend>
+        <div class="form-controls">
+          <label for="comment">
+            Comments:
+          </label>
+          <textarea
+            name="comment"
+            id="comment"
+            cols="20"
+            rows="2"
+          >
+          </textarea>
+        </div>
+        <label for="recommend">
+          Would you recommend us?
+        </label>
+        <select
+          name="recommend"
+          id="recommend"
+        >
+          <option value="yes">yes</option>
+          <option value="no">no</option>
+        </select>
+
+      </fieldset>
+      <label for="submit"><input type="submit" name="submit"></label>
```

Comment:
@MabuzaM Paul mentioned that `<input type="submit"` is considered deprecated - better use `<button type="submit">Submit</button>`
```suggestion
      <button type="submit">Submit</button>
```